### DISCUSSION TITLE
DLL overrides

### DISCUSSION THREAD

**Discussion by dp27thelight** [Heroic-Games-Launcher/HeroicGamesLauncher#3299] [Q&A]
How do you set DLL overrides?

example:

`WINEDLLOVERRIDES="dinput8=n,b"`

I see wrapper arguments and environment variables in ADVANCED section, I'm used to lutris and a bit confused.

**Comment by Giger22** [comment]
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/discussions/2057

**Comment by Recker193** [comment]
It took me like 1-2 days to figure this out, but it's not that hard. Downlaod the WSOCK32.dll into the PAYDAY2 folder, then define an Environment Variable called WINEDLLOVERRIDES and set its value to wsock32=n,b. The last step is to put $WINEDLLOVERRIDES into the Game Arguements, PAYDAY 2 should prompt you to set up WSOCK32 like normal.

**Comment by jsphonorio** [comment]
@Recker193 thank you for following up on your issue. I was trying to get the House of the Dead Remake mod to wrok wich uses UnityDoorStop loader and it usies winhttp.dll as the override to start it. I followed your advice but did winhttp.dll=n,'b and it started running perfectly!
